stdlib/auth/src/main/ballerina/auth/Module.md

```diff
@@ -6,7 +6,11 @@ This module provides a set of default authentication store providers that can be
 
 An authentication store provider defines an authentication scheme that could be used to protect endpoints. The `auth:AuthStoreProvider` type acts as the interface for all the authentication providers. Any type of implementation, such as LDAP, JDBC, and file based, should be object-wise similar. 
 
-By default, there are two implementations of the `auth:AuthProvider`. They are, the `auth:ConfigAuthProvider`, which authenticates based on usernames and passwords stored in a configuration file, and the `auth:JWTAuthProvider`, which authenticates by validating a JWT. It is possible to implement more such authentication mechanisms. 
+By default, there are three implementations of the `auth:AuthStoreProvider`. They are, the `auth:ConfigAuthStoreProvider`,
```

Comment:
Change to:

"They are; the `auth:ConfigAuthStoreProvider`, which authenticates based on usernames and passwords stored in a configuration file, the `auth:JWTAuthProvider`, which authenticates by validating a JWT, and finally the `auth:LdapAuthStoreProvider`, which authenticates based on the user credentials stored in an active directory or an LDAP."